docs/support-guide.md

```diff
@@ -43,21 +43,42 @@ The following conditions require new certification assets:
 
 ## Review Environment <a name="setup"></a>
 
-### Setting up local environment <a name="setup-install"></a>
+### Install Tools <a name="setup-install"></a>
 
 - Download the [openshift-provider-cert](./user.md#install): OpenShift Provider Certification tool
 - Download the [`omg`](https://github.com/kxr/o-must-gather): tool to analyse Must-gather archive
-
 ```bash
 pip3 install o-must-gather --user
 ```
+- Download the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) or another tool to pull from AWS S3
+
+### Download Baseline CI results <a name="setup-download-baseline"></a>
 
-### Download dependencies <a name="setup-download-dependencies"></a>
+The Openshift provider certification tool is run periodically ([source code](https://github.com/openshift/release/blob/master/ci-operator/jobs/redhat-openshift-ecosystem/provider-certification-tool/redhat-openshift-ecosystem-provider-certification-tool-main-periodics.yaml)) in OpenShift CI using the latest stable release of OpenShift. These baseline results are stored long-term in an AWS S3 bucket (`s3://openshift-provider-certification/baseline-results`). These baseline results should be used as a reference when reviewing a partner's certification results.
 
-- Download the Baseline results from the <TBD>`quay.io/ocp-cert/baseline-results`
+You will need and AWS Access Key/ID for the `openshift-dev` AWS account in order to download from the S3 bucket.
```

Comment:
I'm not sure how to provide more specific guidance here (like how to get a access key, etc).

Also, should we consider just making this bucket public since its public on prow.ci.openshift.org already?

Reply:
@bostrt no, the artifacts should be public somewhere. As we are using S3, we should set the Bucket to be able to list it on, ideally, on [the browser](https://docs.aws.amazon.com/AmazonS3/latest/userguide/HostingWebsiteOnS3Setup.html).